arena.js

```diff
@@ -0,0 +1,30 @@
+class Arena{
+  constructor(name){
+      this.name = name.charAt(0).toUpperCase() + name.slice(1);
+      this.gladiators = [];
+  }
+  addFighter(gladiator){
+      this.gladiators.push(gladiator)
+      if((colosseum.gladiators).length > 2){  //set a max of 2 fighters
```

Comment:
Keep in mind this will add a gladiator if you currently have two in the array (i.e., your condition only kicks in when you have 3 or more). Should use `>=`